### PR TITLE
Add build your own pack tile

### PR DESCRIPTION
**Pull request by LizBaker**
## Description

This PR is for adding a tile to the observability pack list that links to the pack repo to encourage folks to build their own

TODO
- [ ] confirm and implement design
- [ ] confirm and implement copy
- [x] adjust instrumentation for `build your own pack` clicks
- [x] decide whether to create a new component or stick with the manipulations to PackTile
      This issue is mainly stemming from the fact that the tile needs to externally link and is currently built for internal links

## Related Issue(s) / Ticket(s)

Closes https://github.com/newrelic/developer-website/issues/1481

### Current WIP
Screen Shot 2021-07-14 at 2 43 22 PM

### CODE REVIEW

File: src/pages/observability-packs.js [hunk 1]
```diff
@@ -351,6 +352,24 @@ const ObservabilityPacksPage = ({ data, location }) => {
...
+          href="https://github.com/newrelic/newrelic-observability-packs"
+          target="_blank"
+          rel="noreferrer"
+          css={css`
+            text-decoration: none;
+          `}
+        >
+          <PackTile
+            css={css`
+              height: 100%;
```
Comment (zstix) [inline src/pages/observability-packs.js:365]: I think we'll need to do something to address the position in the list view (right now it's doing some weird things).

File: src/pages/observability-packs.js [hunk 2]
```diff
@@ -351,6 +352,24 @@ const ObservabilityPacksPage = ({ data, location }) => {
           `}
         `}
       >
+        <a
+          href="https://github.com/newrelic/newrelic-observability-packs"
+          target="_blank"
+          rel="noreferrer"
```
Comment (zstix) [inline src/pages/observability-packs.js:358]: I _think_ we might be able to use the `ExternalLink` component for this. Not sure if it's worth the update, just wanted to mention it.

File: src/components/PackImg.js [hunk 3]
```diff
@@ -34,56 +41,20 @@ const PackImg = ({ className, logoUrl, packName }) => {
...
-          position: relative;
-          top: 80%;
-          left: 83%;
-        `}
-      />
       <p
         css={css`
           text-align: center;
-          line-height: 150%;
+          line-height: ${packAcronym.length < 4 ? '190%' : '300%'};
```
Comment (zstix) [inline src/components/PackImg.js:54]: Hopefully this covers the most common acronym lengths. We might be able to calculate a % based on the acronym length. Another possible option is to just wrap to a new line.

This is fine as-is, I'm just thinking out loud.

**Review by zstix — CHANGES_REQUESTED** [review]
Lookin' good!

The only thing I'd like to see us update is the treatment of the new tile in list-view.

File: src/components/PackImg.js (hunk 3 above)
Comment (aswanson-nr) [reply src/components/PackImg.js]: This was removed in favor of using flexbox for centering

File: src/pages/observability-packs.js (hunk 1 above)
Comment (aswanson-nr) [reply src/pages/observability-packs.js]: This should be fixed now!

**Review by zstix — APPROVED** [review]

**Pull request merged**